pkgs/development/libraries/updfparser/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchgit }:
+
+stdenv.mkDerivation {
+  name = "updfparser";
+  version = "unstable-2022-03-16";
+
+  src = fetchgit {
+      url = "git://soutade.fr/updfparser";
+      rev = "9d56c1d0b1ce81aae4c8db9d99a8b5d1f7967bcf";
+      sha256 = "sha256-9dvibKiUbbI4CrmuAaJzlpntT0XdLvdGeC2/WzjlA5U=";
```

Comment:
Fix the indentation and also the download hashes.

Reply:
I fixed the indentation, but regarding the hashes, it's a bit more complicated than that:

```console
$ nix hash to-sri --type sha256 \
  `nix-prefetch-git git://soutade.fr/updfparser \
    9d56c1d0b1ce81aae4c8db9d99a8b5d1f7967bcf \
    2> /dev/null | jq -r .sha256`
sha256-9dvibKiUbbI4CrmuAaJzlpntT0XdLvdGeC2/WzjlA5U=
```

I am clueless why the CI got different values.